prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -845,6 +845,6 @@ class Hl7Serializer(val metadata: Metadata): Logging {
         const val MESSAGE_TRIGGER_EVENT = "R01"
         const val SOFTWARE_VENDOR_ORGANIZATION: String = "Centers for Disease Control and Prevention"
         const val SOFTWARE_PRODUCT_NAME: String = "PRIME Data Hub"
-        val HD_FIELDS = listOf("MSH-4-1", "OBR-3-2", "OBR-2-2", "ORC-3-2", "ORC-2-2")
+        val HD_FIELDS = listOf("MSH-4-1", "OBR-3-2", "OBR-2-2", "ORC-3-2", "ORC-2-2", "PID-3-4-1")
     }
```

Comment:
In order to make sure the name truncates properly, I need to add it here. I don't think HAPI has a good way to find and truncate all HD fields. We'd probably need to build an extension method to do it.